src/Spice86.Core/Emulator/VM/Machine.cs

```diff
@@ -54,23 +53,13 @@ public sealed class Machine : IDisposable {
     /// <summary>
     /// The emulated CPU.
     /// </summary>
-    public Cpu Cpu { get; }
```

Comment:
Don't remove properties (State, Stack) inside the Machine object, they are there to be used by C# overrides of emulated program code.

Reply:
Its not removed but replaced with ICpu. Existing overrides will break because the CPU was exposed directly before.